### PR TITLE
Readme for operators and running the program

### PR DESCRIPTION
**Pull request by BenBals**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,36 @@ truthtable reads in a logical expression and generates from that a LaTeX tabular
...
+- negation
+  - `not`, `!`, `\neg`, `\lnot`
+- xor
+  - `xor`, `^`, `\oplus`, `\veebar`
+- implication
+  - `>`, `\imp`, `\rightarrow`, `\implies`
+- equivalence
+  - `=`, `\aeq`, `\leftrightarrow`
+  
+Please note, that the representation you input will be used in the output.
```
Comment (felixrindt) [inline README.md]: Please change to `Please note that the output will use the same token representation as your input.`

File: README.md [hunk 2]
```diff
@@ -16,8 +16,36 @@ truthtable reads in a logical expression and generates from that a LaTeX tabular
...
+
+## Running the program. 
+
+The expression for which the truthtable is to be build is input via one of two ways:
+1. the standard input
```
Comment (felixrindt) [inline README.md]: Should be `The first line of standard input`.

File: README.md [hunk 3]
```diff
@@ -16,8 +16,36 @@ truthtable reads in a logical expression and generates from that a LaTeX tabular
...
+```
+$ python3 truthtable.py
+a or b
+[output is printed...]
+```
+2. a file
+```
+$ python3 truthtable.py examples/1.txt
+[output is printed...]
+```
```
Comment (felixrindt) [inline README.md:54]: There's a third way: command line argument `python3 truthtable.py "a or b"`

**Review by felixrindt — CHANGES_REQUESTED** [review]

**Comment by felixrindt** [comment]
Thanks for all your PRs :)

**Comment by BenBals** [comment]
No problem, thanks for the nice program!

I included your requested changes.

File: README.md [hunk 4]
```diff
@@ -6,18 +6,51 @@ truthtable reads in a logical expression and generates from that a LaTeX tabular
...
+- implication
+  - `>`, `\imp`, `\rightarrow`, `\implies`
+- equivalence
+  - `=`, `\aeq`, `\leftrightarrow`
+  
+Please note that the output will use the same token representation as your input.
+
+## Running the program. 
+
+The expression for which the truthtable is to be build is input via one of two ways:
```
Comment (felixrindt) [inline README.md]: one of three ways then ;-)

**Review by felixrindt — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (BenBals) [reply README.md]: This should be fixed in my latest commit.

**Review by felixrindt — APPROVED** [review]

**Pull request merged**